public/app/features/alerting/unified/hooks/useExternalAMSelector.test.tsx

```diff
@@ -197,6 +197,26 @@ describe('useExternalDataSourceAlertmanagers', () => {
   });
 });
 
+describe('normalizeDataSourceURL', () => {
+  it('should add "http://" protocol if missing', () => {
+    const url = 'example.com';
+    const normalizedURL = normalizeDataSourceURL(url);
+    expect(normalizedURL).toBe('http://example.com');
+  });
+
+  it('should not modify the URL if it already has a protocol', () => {
+    const url = 'https://example.com';
+    const normalizedURL = normalizeDataSourceURL(url);
+    expect(normalizedURL).toBe(url);
+  });
+
+  it('should remove trailing slashes from the URL', () => {
+    const url = 'http://example.com/';
```

Comment:
Should we also remove multiple trailing slashes, for good measure? e.g. 'http://example.com////'